p2p/pack.go

```diff
@@ -66,40 +66,45 @@ func UnPackRegisterData(parameters [][]byte) (nodeType string, id peerlib.ID, ad
 }
 
 //PackRegisterData pack node message into p2pMessage
-func PackRegisterData(chain, fn string, nodeType string, id peerlib.ID, addrs []ma.Multiaddr, ts time.Time) ([][]byte, error) {
+func PackRegisterData(chain, fn string, nodeType string, id peerlib.ID, addrs []ma.Multiaddr, ts time.Time) [][]byte {
 	typePacked := []byte(nodeType)
 	idPacked, err := id.Marshal()
```

Comment:
Since there will not be an error in this function, we can use `idPacked, _ := id.Marshal()` without checking its error.

Reply:
I have a second thought about this. When someone reads the code, would this person curious about why?  Would leave the error here cause less confusion?